### PR TITLE
Fix some test coverage misses

### PR DESCRIPTION
**Pull request by nmajor25**
Shortcut Story ID: [sc-34796]

### CODE REVIEW

File: src/js/views/forms/form/form_views.js [hunk 1]
```diff
@@ -34,9 +34,7 @@ const ContextTrailView = View.extend({
     'click .js-dashboard': 'click:dashboard',
   },
   onClickBack() {
-    Radio.request('history', 'go:back', () => {
-      this.routeToPatient();
-    });
```
Comment (nmajor25) [inline src/js/views/forms/form/form_views.js]: Pertaining to this coverage miss: https://coveralls.io/builds/58331475/source?filename=src%2Fjs%2Fviews%2Fforms%2Fform%2Fform_views.js#L38.

Does `this.routeToPatient()` do anything here? Testing in my local sandbox, this behaves the same without it.

If that can be removed here, we should be able to change `onClickDashboard()` below to this:

```js
onClickDashboard() {
  Radio.trigger('event-router', 'patient:dashboard', this.patient.id);
},
```

And get rid of the `routeToPatient()` function.

File: test/integration/patients/worklist/worklist.js
Comment (nmajor25) [inline test/integration/patients/worklist/worklist.js:1]: To add coverage for this: https://coveralls.io/builds/58331475/source?filename=src%2Fjs%2Fapps%2Fpatients%2Fsidebar%2Ffilters-sidebar_app.js#L40. 

That code/feature was added here: https://github.com/RoundingWell/care-ops-frontend/pull/935/commits/b82e7ab24919f0afd2bd5b89268bb86426b9b886.

I also added the same changes to the schedule version of this test.

File: test/integration/patients/worklist/worklist.js [hunk 2]
```diff
@@ -267,6 +271,20 @@ context('worklist page', function() {
...
+
+    cy
+      .url()
+      .should('contain', 'patient/dashboard/1')
+      .wait('@routePatient');
+
+    cy
+      .go('back')
+      .wait('@routeFlows');
+
```
Comment (nmajor25) [inline test/integration/patients/worklist/worklist.js:287]: To add coverage for this: https://coveralls.io/builds/58331475/source?filename=src%2Fjs%2Fviews%2Fpatients%2Fworklist%2Fflow_views.js#L76.

Don't really understand how this line was missing coverage. But I'm guessing it was from when the default actions vs. flows toggle was changed for the worklist tests.

File: src/js/views/forms/form/form_views.js (hunk 1 above)
Comment (paulfalgout) [reply src/js/views/forms/form/form_views.js]: This was added so if you hard refreshed on a form and did not have any previous history, the back button would go to the patient.  It's needed.. I dunno how we'd actually test it though.

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: src/js/views/forms/form/form_views.js (hunk 1 above)
Comment (nmajor25) [reply src/js/views/forms/form/form_views.js]: Ah okay, I just added that back.

`istanbul ignore`?

Comment (paulfalgout) [reply src/js/views/forms/form/form_views.js]: actually now that I think about it, it's testable.. just `visit` a patient form and push go back.. it should go to that patient's dashboard even though that was never a route.

Comment (nmajor25) [reply src/js/views/forms/form/form_views.js]: Just added that in my last push.

**Review by paulfalgout — APPROVED** [review]

**Pull request merged**